lib/modules/manager/npm/update/dependency/pnpm.spec.ts

```diff
@@ -0,0 +1,336 @@
+import { codeBlock } from 'common-tags';
+import * as npmUpdater from '../..';
+
+/**
+ * Per the YAML spec, a document ends with a newline. The 'yaml' library always
+ * uses that when serialising, but `codeBlock` strips the last indentation. This
+ * helper makes assertions simpler.
+ */
+function yamlCodeBlock(
+  literals: TemplateStringsArray,
+  ...placeholders: any[]
+): string {
+  return codeBlock(literals, placeholders) + '\n';
+}
```

Comment:
If this is the case it should be moved to `lib/util/yaml`

Reply:
no, it's only used for tests, so should be somewhere in `test/...`

wonder why it's needed at all 🤔 we use code block in other tests with yaml